### ISSUE TITLE
Is it neccessary to perform the target scene finetuning?

### ISSUE THREAD

**Issue by ludekcizinsky** [GSFix3D/GSFix3D#3]
Hi,
thank you for the great work. From the paper, I was under the impression that you only finetune the SDv2 on synthetic datasets, and then use this finetuned model to directly fix the novel views for Scannet++ and Replica. However, from your instructions here in Readme, it seems that you can also finetune on the target scene. So I was wondering if the results you reported in the paper also used this per scene finetuning?

**Comment by goldoak** [comment]
Hi, Thanks for your question! To be clear, **all of the key results reported in our paper, including those for ScanNet++ and Replica, were achieved using the per-scene fine-tuning step.** We never hinted anywhere in the paper that we "_only fine-tuned on synthetic datasets_". We have a whole Section 3.1 discussing our proposed fine-tuning protocol, which is the core contribution of our method. The pre-training on synthetic datasets serves as a warm-up phase to adapt the Stable Diffusion model to the novel input layers and the repair task. The critical performance improvement comes from the final per-scene fine-tuning. You can also find a detailed comparison, including the results of DiFiX fine-tuned per-scene, in Section 7.1 of our supplementary material.

<img width="576" height="265" alt="Image" src="https://github.com/user-attachments/assets/e8dbbdd8-e261-44e6-81c9-7846b959db9e" />
<img width="576" height="306" alt="Image" src="https://github.com/user-attachments/assets/81dd321b-a886-4425-bdbe-993e718a1fb1" />

**Comment by ludekcizinsky** [comment]
Thank you for the detailed answer. It makes sense now.
